app/constants/document-type-enum.js

```diff
@@ -1,3 +1,9 @@
 module.exports = {
-  VISIT_CONFIRMATION: 'VISIT-CONFIRMATION'
+  'VISIT_CONFIRMATION': 'VISIT-CONFIRMATION',
+  'working-tax-credit': 'WORKING-TAX-CREDIT',
+  'pension-credit': 'PENSION-CREDIT',
+  'hc2': 'HC2',
+  'hc3': 'HC3',
+  'nhs-tax-credit': 'NHS-TAX-CREDIT',
+  'RECEIPT': 'RECEIPT'
```

Comment:
use benefit name in lower case so we don't need to map benefit to documenttype in claim summary or internal to find document